Bottom: ddbb4a7e24675ea08e1c7668fabdd6c39268d374
Top:    8e4c634570393da66f81a9cb4467977886b8ba20
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2012-12-13 02:02:15 +0000

Refresh of bentley-ottann-test-implementa

---

diff --git a/src/hid/common/hidgl.c b/src/hid/common/hidgl.c
index 8b09e8b..4f8ac1a 100644
--- a/src/hid/common/hidgl.c
+++ b/src/hid/common/hidgl.c
@@ -624,7 +624,7 @@ fill_polyarea (POLYAREA *pa, const BoxType *clip_box, double scale)
   borast_traps_t traps;
 
   _borast_traps_init (&traps);
-  bo_poly_to_traps (poly->Clipped, &traps);
+  bo_poly_to_traps (pa, &traps);
   _borast_traps_fini (&traps);
 
   return;
